spec/01-unit/08-router_spec.lua

```diff
@@ -4853,3 +4856,112 @@ for _, flavor in ipairs({ "traditional", "traditional_compatible" }) do
     end)
   end)
 end
+
+do
+  local flavor = "expressions"
+
+  describe("Router (flavor = " .. flavor .. ")", function()
+    reload_router(flavor)
+
+    local use_case, router
+
+    lazy_setup(function()
+      use_case = {
+        -- query has one value
+        {
+          service = service,
+          route   = {
+            id = "e8fb37f1-102d-461e-9c51-6608a6bb8101",
+            expression = [[http.path == "/foo/bar" && http.queries.a == "1"]],
+            priority = 100,
+          },
+        },
+        -- query has no value or is empty string
+        {
+          service = service,
+          route   = {
+            id = "e8fb37f1-102d-461e-9c51-6608a6bb8102",
+            expression = [[http.path == "/foo/bar" && http.queries.a == ""]],
+            priority = 100,
+          },
+        },
+        -- query has multiple values
+        {
+          service = service,
+          route   = {
+            id = "e8fb37f1-102d-461e-9c51-6608a6bb8103",
+            expression = [[http.path == "/foo/bar" && any(http.queries.a) == "2"]],
```

Comment:
What does `any` mean here? I think it means "If any value of the query matches, the whole query matches". Do we have documents to explain the transformers?

Reply:
1. `any(field)` means that one value of this field  matches the predicate.
2.  now we have not documentation for `any`, sorry.